{Thu 14:48:50 Step:1779} Server_Comm.py: Received a new packet while on world.update()!
